gef.py

```diff
@@ -2755,7 +2755,10 @@ def get_os():
 @lru_cache()
 def get_pid():
     """Return the PID of the debuggee process."""
-    return gdb.selected_inferior().pid if not __gef_qemu_mode__ else gdb.selected_thread().ptid[1]
+    pid = gdb.selected_inferior().pid if not __gef_qemu_mode__ else gdb.selected_thread().ptid[1]
+    if pid == 0:
+        raise RuntimeError("cannot retrieve PID for debuggee process")
```

Comment:
I wonder whether we should use 'interior' as is standard in gdb parlance. @hugsy what do you think?

Reply:
what do you mean by using `interior`?

Reply:
I mean `inferior` It GDB it means the target, e.g. `info inferior`: https://moss.cs.iit.edu/cs351/gdb-inferiors.html

I think `target` or `inferior`. Either way we should be consistent.

Reply:
ah, you mean the error message, right? i was wondering about the `pid = ...` line.

yeah, you are right, it should be more consistent.

Reply:
Yeah, I am nit-picky about consistency and stuff :) but that's why I commented on the `raise` line.

Reply:
let's use `target process` and fix in the doc string.